build/helpers/coverage.js

```diff
@@ -24,19 +24,18 @@ const session = new Session();
 
 const metrics = [
   {
-    ...Arithmetic,
-    weight: 1 / 3
+    ...Arithmetic
   },
   {
-    ...Byte,
-    weight: 1 / 3
+    ...Byte
   },
   {
-    ...Logical,
-    weight: 1 / 3
```

Comment:
Since you have removed the second element of the metric-object, this can be reduced to something like `const metrics = [Arithmetic, Byte, Logical];`